doc/connectivity/connectome_extraction.rst

```diff
@@ -0,0 +1,232 @@
+.. _connectome_extraction:
+
+============================================================================
+Connectome extraction: inverse covariance for direct connections 
+============================================================================
+
+.. topic:: **Page summary**
+
+   Given a set of time-series (eg as extracted in the previous section)
+   A *functional connectome* is a set of connections representing brain
+   interactions between regions. Here we show the use of sparse-inverse
+   covariance to extract functional connectomes focussing only on direct
+   interactions between regions.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+.. topic:: **References**
+
+   * `Smith et al, Network modelling methods for FMRI,
```

Comment:
Generell: I learned that it always should be "et al.", that is, with the point at the end (=et alii)

http://fr.wikipedia.org/wiki/Et_al.